src/app/features/register-student/register-student.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="container">
+    <div class="register-form-students">
+        <form class="form-students" [formGroup]="formGroup">
+            <div class="example-container">
+                <h1 class="title"> Registro de Estudiantes</h1>
+                <mat-form-field class="seccion-register"appearance="fill">
+                    <mat-label>Identificación Estudiante</mat-label>
+                    <input matInput name="idStudent" formControlName="idStudent" type="number"
+                    placeholder="Identificación"  required>
+                  <mat-error *ngIf="formGroup.get('idStudent').hasError('required')">Es necesario llenar el campo</mat-error>
+                </mat-form-field>
+                <mat-form-field appearance="fill">
+                    <mat-label>Nombre Estudiante</mat-label>
+                    <input matInput name="nameStudent" formControlName="nameStudent" type="text"
+                    placeholder="Nombre Estudiante"  required>
+                  <mat-error *ngIf="formGroup.get('nameStudent').hasError('required')">Es necesario llenar el campo</mat-error>
+                </mat-form-field>
+                <mat-form-field appearance="fill">
+                    <mat-label>Apellido Estudiante</mat-label>
+                    <input matInput name="lastNameStudent" formControlName="lastNameStudent" type="text"
+                    placeholder="Apellido Estudiante"  required>
+                  <mat-error *ngIf="formGroup.get('lastNameStudent').hasError('required')">Es necesario llenar el campo</mat-error>
+                </mat-form-field>
+                <mat-form-field appearance="fill">
+                    <mat-label>Grado</mat-label>
+                    <select name="grade" formControlName="grade" matNativeControl required>
+                        <option class="option" value="prejardin">Pre-Jardín</option>
+                        <option class="option" value="jardin">Jardín</option>
+                        <option class="option" value="transicion">Transición</option>
+                        <option class="option" value="primero">Primero</option>
+                        <option class="option"  value="segundo">Segundo</option>
+                        <option class="option" value="tercero">Tercero</option>
+                        <option class="option" value="cuarto">Cuarto</option>
+                        <option class="option" value="quinto">Quinto</option>
+                    </select>
+                    <mat-error *ngIf="formGroup.get('grade').hasError('required')">Es necesario llenar el campo</mat-error>
+                </mat-form-field>
+                <mat-form-field appearance="fill">
+                    <mat-label>Tipo Estudiante</mat-label>
+                    <select name="tipe" formControlName="tipe" matNativeControl required>
+                        <option class="option" value="Normal">Normal</option>
```

Comment:
use lowercase for values, only titlecase for Text
```suggestion
                        <option class="option" value="normal">Normal</option>
```